src/Offer/AbstractGeoCoordinatesCommandHandler.php

```diff
@@ -23,12 +23,12 @@ abstract class AbstractGeoCoordinatesCommandHandler extends Udb3CommandHandler i
     private $offerRepository;
 
     /**
-     * @var AddressFormatterInterface
+     * @var AddressFormatter
      */
     private $defaultAddressFormatter;
```

Comment:
```suggestion
    private AddressFormatter $defaultAddressFormatter;
```

Reply:
I only did a rename to remove the `Interface` suffix, but did not clean up any typehints of this interface or its implementations in existing code because that felt like it would become a never-ending change. Because then the next step would be to clean up the other typehints in those classes that I touched to clean up the `AddressFormatter` typehints, etc etc. I believe that it would be better to clean up those classes like `AbstractGeoCoordinatesCommandHandler` later in separate PRs and clean up all of their types and do other boyscouting in those files in one go, and only do boyscouting on code that is at most "1 step removed" from the changes in scope of this PR.

Reply:
Alrighty 👍🏻